### PR TITLE
Safer unlocking with descriptions

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: paddles/models/nodes.py [hunk 1]
```diff
@@ -120,6 +120,11 @@ def _check_for_update(self, values):
                 raise ForbiddenRequestError(
                     "Cannot {verb} - locked_by values must match".format(
                         verb=verb))
+            elif (verb == 'unlock' and was_locked and desc and desc !=
```
Comment (andrewschoen) [inline paddles/models/nodes.py]: I don't think you need the extra 'and desc' here.  Wouldn't it be set to None above? So, None != "some description" would be fine.

File: paddles/tests/controllers/test_nodes.py [hunk 2]
```diff
@@ -349,6 +349,18 @@ def test_unlock_different_owner(self):
             dict(locked=False, locked_by='someone@else'), expect_errors=True)
         assert response.status_int == 403
 
+    def test_unlock_different_description(self):
```
Comment (andrewschoen) [inline paddles/tests/controllers/test_nodes.py]: Would a test where description is None be helpful?

File: paddles/models/nodes.py (hunk 1 above)
Comment (andrewschoen) [inline paddles/models/nodes.py]: On second thought, if we removed the extra 'and desc' here would it keep people from unlocking who locked without a description?

Comment (zmc) [inline paddles/models/nodes.py]: That's correct; it's why it's in there.

File: paddles/tests/controllers/test_nodes.py (hunk 2 above)
Comment (zmc) [inline paddles/tests/controllers/test_nodes.py]: I think that's covered by ``test_unlock()`` and ``test_unlock_different_owner()``

**Pull request merged**